CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - The project Golang version is updated from the end-of-life v1.12 to the latest
   version v1.15.
   [cyberark/conjur-authn-k8s-client#206](https://github.com/cyberark/conjur-authn-k8s-client/issues/206)
+- Improve the error message raised when the username doesn't include the `host/` prefix
```

Comment:
changes should go at the top, no?

Reply:
i don't think that the order within every section is important.

Reply:
all the content is in the same release

Reply:
ok, I thought it does.